src/components/Textarea/index.js

```diff
@@ -135,6 +137,9 @@ class Textarea extends Component {
                 <RenderIf isTrue={!!error}>
                     <StyledError id={this.getErrorMessageId()}>{error}</StyledError>
                 </RenderIf>
+                <RenderIf isTrue={!!footer}>
+                    <StyledFooter>{footer}</StyledFooter>
+                </RenderIf>
```

Comment:
@TahimiLeonBravo the footer position is before the bottomHelpText and the error?